doc/TSCScope.md

```diff
@@ -0,0 +1,31 @@
+Subject to the contraints defined in the foundation
+charter, these are the areas that the TSC has autonomy
+in making decisions (some of which can be delegated to
+other groups, for example the CTC).  Outside of these
+areas the TSC will collaborate with the board
+when making decisions:
+
+* Code/Doc/ changes to any of the repositories under github/nodejs
+* Standards and processes covering contributions to repositories under
```

Comment:
Yes.  The TSC delegates some of its autonomy to the Working Groups but it is through the TSC that this autonomy originates.